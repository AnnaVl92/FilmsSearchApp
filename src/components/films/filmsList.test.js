import React from 'react';
import FilmsList from './filmsList.jsx';
import {shallow} from "enzyme";
import toJson from "enzyme-to-json";

it('renders correctly', () => {
	const mockFilms = [{
			id: 1,
			img: "https://vignette.wikia.nocookie.net/cinemorgue/images/9/98/Kill_bill2_01.jpg/revision/latest?cb=20151221212736",
			title: "Kill Bill: Volume 2",
			releaseDate: 2004,
			genre: "Action & Adventure"
		},
		{
			id: 2,
			img: "https://vignette.wikia.nocookie.net/immwdb/images/e/e4/Kill-bill-vol-1.jpg/revision/latest?cb=20150116205604",
			title: "Kill Bill: Volume 1",
			releaseDate: 2003,
			genre: "Action & Adventure"
		}]
	;

	const component = shallow(<FilmsList films={mockFilms} />);
	expect(component).toMatchSnapshot();
});