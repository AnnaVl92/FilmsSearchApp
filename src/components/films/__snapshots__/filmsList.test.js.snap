// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="row"
>
  <Film
    film={
      Object {
        "genre": "Action & Adventure",
        "id": 1,
        "img": "https://vignette.wikia.nocookie.net/cinemorgue/images/9/98/Kill_bill2_01.jpg/revision/latest?cb=20151221212736",
        "releaseDate": 2004,
        "title": "Kill Bill: Volume 2",
      }
    }
    key="1"
  />
  <Film
    film={
      Object {
        "genre": "Action & Adventure",
        "id": 2,
        "img": "https://vignette.wikia.nocookie.net/immwdb/images/e/e4/Kill-bill-vol-1.jpg/revision/latest?cb=20150116205604",
        "releaseDate": 2003,
        "title": "Kill Bill: Volume 1",
      }
    }
    key="2"
  />
</div>
`;
